examples/ex1.cpp

```diff
@@ -82,6 +87,8 @@ int main(int argc, char *argv[])
                   "--no-static-condensation", "Enable static condensation.");
    args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
                   "--no-partial-assembly", "Enable Partial Assembly.");
+   args.AddOption(&amgx, "-amgx", "--amgx-precon", "-no-amgx",
+                  "--no-amgx-precon", "Use AmgX V-cycle as preconditioner for CG.");
```

Comment:
I'm not sure if this was discussed before: since we have a separate set of `ex1`/`ex1p` in the `amgx` sub-directory, isn't it better to keep the regular `ex1`/`ex1p` simpler and remove the AmgX options from them?